src/events/http2.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @fileoverview Handlers for http & https libraries instrumentation
+ */
+
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+const urlLib = require('url');
+const http2 = require('http2');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+const { isBlacklistURL, isBlacklistHeader } = require('.././helpers/events');
+const {
```

Comment:
might be better to patch both in the same file? how different are they?

Reply:
The tracing has some very significant difference both in the patching and in the wrapping, so I wouldn't recommend it.
I did export common HTTP functions from http.js